tests/functional/cylc-ping/04-check-keys-remote.t

```diff
@@ -37,19 +37,23 @@ __FLOW_CONFIG__
 run_ok "${TEST_NAME_BASE}-validate" cylc validate "${SUITE_NAME}" \
     -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
 suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}" \
-    -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
+    --debug -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
 RRUND="cylc-run/${SUITE_NAME}"
 RSRVD="${RRUND}/.service"
 poll_grep_suite_log 'Holding all waiting or queued tasks now'
 SSH='ssh -n -oBatchMode=yes -oConnectTimeout=5'
+ 
 ${SSH} "${CYLC_TEST_HOST}" \
 find "${RSRVD}" -type f -name "*key*"|awk -F/ '{print $NF}'|sort >'find.out'
-cmp_ok 'find.out' <<__OUT__
+sort >'keys' <<__OUT__
+client_${CYLC_TEST_INSTALL_TARGET}.key
 client.key_secret
-client_$CYLC_TEST_PLATFORM.key
 server.key
```

Comment:
I don't think this sort is necessary, as you're sorting `client.*` and `client_*` which should always have the same order anyway

Reply:
I have removed the sort. The test passes locally but fails GH Actions: 
![image](https://user-images.githubusercontent.com/37735232/106262581-2b98b800-621b-11eb-909c-5fe5f87b1d6b.png)

Putting the sort back fixes this I think.